pkg/acquisition/file_reader.go

```diff
@@ -113,12 +100,12 @@ func InitReader(cfg string) (*FileAcquisCtx, error) {
 				log.Tracef("End of yaml file")
 				break
 			}
-			log.Fatalf("Error decoding acquisition configuration file with '%s': %v", cfg, err)
+			log.Fatalf("Error decoding acquisition configuration file with '%s': %v", config.Crowdsec.AcquisitionFilePath, err)
 			break
 		}
 		files = append(files, t)
 	}
-	return InitReaderFromFileCtx(files)
+	return files, nil
```

Comment:
Am I mistaken if I say that files nil there ?

Reply:
it returns `([]FileCtx, error)`